db/migrate/20161118210435_add_question_number.rb

```diff
@@ -0,0 +1,6 @@
+class AddQuestionNumber < ActiveRecord::Migration
+  def change
+    add_column :questions, :number, :integer, unique: true
```

Comment:
Now that I look at the schema, there's not an easy way to put a unique constraint on (survey_id, number) because the question table doesn't store survey_id. Will remove for now and think about this some more.